backend/app/views/spree/admin/orders/customer_details/edit.html.erb

```diff
@@ -1,11 +1,10 @@
 <%= render :partial => 'spree/admin/shared/order_tabs', :locals => { :current => 'Customer Details' } %>
 
 <% content_for :page_title do %>
-  <i class="fa fa-arrow-right"></i> <%= Spree.t(:customer_details) %>
+  &#x2f; <%= Spree.t(:customer_details) %>
 <% end %>
 
 <% content_for :page_actions do %>
```

Comment:
Oh, my thought was that without some text in that `content_for` that then there would no longer be a `page_actions` element on the page and possibly people would be defacing that but I haven't checked this hypothesis.